common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -278,7 +278,7 @@ public double getDouble(String key) {
     throw new RuntimeException("Invalid configuration key " + key + ".");
   }
 
-  private float getFloat(String key, final float defaultValue) {
+  private float getFloat(String key) {
```

Comment:
Does this actually ever get called? This is private and I can't find it in this file.